script/proxies/DeployAndSetUpInverterBeacon_v1.s.sol

```diff
@@ -43,7 +43,7 @@ contract DeployAndSetUpInverterBeacon_v1 is Script {
             );
 
             // Register Metadata at the ModuleFactory_v1
-            // @todo - needs changing for new registerMetadata flow
+            // ::@todo - needs changing for new registerMetadata flow -> @marvinkruse This is not ideal at least for the initial setup in case we need it to be repeatable. We might wanna include a Metadata registration array in the factory itself to allow multiple registrations at the same time during init. thoughts?
```

Comment:
Yeah that might be a good idea, I'm taking notes and will implement that once I touch the whole deployment thing (probably Friday or early next week). Will reference it here then to get this comment cleared out and post my solution.